src/compoz.ts

```diff
@@ -342,21 +399,31 @@ export class Compoz {
 		}
 
 		this.elInput.onkeydown = e => {
-			if ((e.ctrlKey || e.metaKey) && e.keyCode === this.keyCodeZ) {
-				document.execCommand("undo")
+			if (e.ctrlKey || e.metaKey) {
+				if (this.keyCodeY) {
```

Comment:
Should this be `e.keyCode == this.keyCodeY`, no?

Reply:
Oh ya, I forgot to compare it with the e.keyCode